docs/topics/install/docker.rst

```diff
@@ -140,6 +135,81 @@ migrations::
 Gotchas!
 ~~~~~~~~
 
+Here's a list of a few of the issues you might face when using docker.
+
+Can't access the web server?
+----------------------------
+
+Check you've created an hosts file entry pointing ``olympia.dev`` to the
+relevant ip address.
+
+Also make sure you've copied ``docker-compose.override.yml.tmpl`` to
+``docker-compose.override.yml`` to get the default ports. If you haven't
+stop the containers with ``docker-compose stop`` copy the file and restart
+with ``docker-compose up -d``.
+
+Another tip is to use ``docker-compose ps`` to check the status of the
+containers. If they are failing to start you should be able to tell here.
+
+Another way to find out what's wrong is to run ``docker-compose logs``.
+
+Getting "Programming error [table] doesn't exist"?
+--------------------------------------------------
+
+Check you've run the ``make initialize_docker`` step.
+
+
+Port collisions (nginx container fails to start)
+------------------------------------------------
+
+Since by default the docker-compose file exposes the port to the nginx
```

Comment:
This sentence is a bit hard to follow. What about:

```
If you're already running a service on port 80 on your host machine, the `nginx` container will fail to start. This is because the docker-compose file tells `nginx` to listen on port 80 by default.
```